src/pybind/mgr/smb/fs.py

```diff
@@ -54,9 +58,7 @@ class CephFSPathResolver:
     map to real paths in the cephfs volume and determine if those paths exist.
     """
 
-    def __init__(
-        self, mgr: Module_T, *, client: Optional[CephfsClient] = None
-    ) -> None:
+    def __init__(self, mgr: Module_T, *, client: Optional[CephfsClient] = None) -> None:
```

Comment:
Did your IDE do this? If so, can you either set your ide to use `black` formatting with the settings we have in our tox.ini file or make sure you run `format-black` from tox before commiting?

Reply:
yeah probably inbuilt linter changed that, let me fix that